### DISCUSSION TITLE
itron 40gb ert experiences

### DISCUSSION THREAD

**Discussion by kpnobvious** [bemasher/rtlamr#229] [General]
Hi,
I've read thru the other comments about the itron 40g (#103 #170 ) and i think i'm in the same boat as mine is in wake up mode due to battery life.

Does anyone with one of these series had luck with an alternative monitoring system for gas usage or find they do actually provide output a bit more frequently?  I guess on the plus side my meter is almost 20 years old, so hopefully it will get replaced with a model that broadcasts more frequently.  The only alternative I've thought of is my own pulse count meter.
![image](https://user-images.githubusercontent.com/78876891/147290469-ae191dd8-031b-4f26-8c93-f8ee1e0d1d76.png)

**Comment by bemasher** [comment]
I previously had a 40G and never did end up receiving any messages from it. Mine was eventually replaced about a year ago, and it transmits every 5-15 minutes.

**Reply by kpnobvious** [reply]
thanks!

**Comment by makeitworktech** [comment]
I am in the same boat - I have a 40GB. Have you had any luck since? My guess is that, when the gas company comes by to read the meter, I should be able to pick up the data?

**Reply by kpnobvious** [reply]
i monitored it for about a month and the gas company did come by, i did not pick up a reading.

**Reply by makeitworktech** [reply]
Ah ok, you were using SCM or SCM+?

Also, did you end up using a pulse count meter?

**Reply by makeitworktech** [reply]
OK, so if anyone else is looking for this information... I got it to work!
I have an Itron 40GB, and I used SCM. It only picks up the meter reading when the truck comes by to read it.

**Reply by kpnobvious** [reply]
That's great.  I thought i was running it both in SCM and SCM+ mode, i was getting everyone else's meters, just not mine.  I'll have to try it again around the read date.  Is your meter also from 2007 or newer?  I did not making a pulse count meter, was messing with this for a bit - https://george.czabania.com/2020_03_16/

**Reply by hak715** [reply]
I have an Itron 40GB meter, and currently not picking anything up using SCM protocol. I am thinking I will have to wait for the truck to come by. What did you use for the meter ID? Is it just the number under the bar code? Do you include "02" before the number? I don't want to wait a month before realizing I have the wrong ID. 
![IMG_0945](https://user-images.githubusercontent.com/42194241/211819724-8e9bc62e-6f2d-472c-a30a-e56f3aa3b455.jpeg)

**Reply by makeitworktech** [reply]
I have to admit I stopped trying to pick up readings after the first one because I had another project I wanted to use the antenna before, but here is the meter config I used. I plan to start using this again. I was trying in the summer, and wasn't using much gas, but now is a different story.
```
- id: 65087065
  protocol: scm
  name: gas_meter
  friendly_name: Gas Meter
  format: "######"
  unit_of_measurement: ft³
  icon: mdi:gas_canister
  device_class: gas
  state_class: total_increasing
```

**Reply by hak715** [reply]
Thanks for sharing. I am using my antenna to pick up my water meter, works perfectly. I am using the same configuration you are using for my gas meter (with my ID), but no luck yet. I am thinking it only puts out a signal when the utility comes to read it.

**Comment by tbitson** [comment]
I have the same meter. Due to the small battery size it only transmits when there is significant gas usage to report.  In the past 2 weeks I only captured 1 transmission packet.- TimOn Jan 11, 2023, at 6:36 AM, hak715 ***@***.***> wrote:﻿
I have an Itron 40GB meter, and currently not picking anything up using SCM protocol. I am thinking I will have to wait for the truck to come by. What did you use for the meter ID? Is it just the number under the bar code? Do you include "02" before the number? I don't want to wait a month before realizing I have the wrong ID.
